### ISSUE TITLE
Admin Inferface shows Update available but logfile is already new version

### ISSUE THREAD

**Issue by pkuehnel** [bassmaster187/TeslaLogger#82]
Admin interface shows 
Teslalogger: 1.31.1.0 Update available: 1.31.5.0

But in the Logfile the startup shows already TeslaLogger Version: 1.31.5.0

Already tried:
 -Replacing the *.exe in the share and reboot does not help
- clicking Update Button in Admin intreface does not help.

(Problem in the current Raspberry PI image)

**Comment by bassmaster187** [comment]
Does this issue still persists?

**Comment by pkuehnel** [comment]
Yes. I did not reinstall the image but meanwhile I upgraded to 1.32.4.0.

Updates in the admin panel are not possible, I only can replace the *.exe file in the SMB share and reboot, then the version in the logfile is updated.

Probably anything goes wrong there because my car is not assigned to my Tesla account already?

I will get my car on Saturday next week and can test again...

**Comment by bassmaster187** [comment]
Updates are only possible if a car is assigned to your tesla account.
